scripts/unzip-archive.ps1

```diff
@@ -0,0 +1,14 @@
+﻿[CmdletBinding()]
+param(
+    [string]
+    $Source,
+
+    [string]
+    $Destination
+)
+
+Write-Verbose "Loading System.IO.Compression.FileSystem Assembly"
+[System.Reflection.Assembly]::LoadWithPartialName('System.IO.Compression.FileSystem')
+
+Write-Verbose "Unpacking $Source to $Destination"
+[System.IO.Compression.ZipFile]::ExtractToDirectory($Source, $Destination)
```

Comment:
oh eclint, why do you refuse to catch this?

Reply:
😭

Reply:
fixed